docs/sources/get-started/_index.md

```diff
@@ -0,0 +1,78 @@
+---
+description: Get started with Writers' Toolkit
+menuTitle: Get started
+title: Get started with Writers' Toolkit
+weight: 50
+aliases:
+  - /docs/writers-toolkit/get-started/
+---
+
+# Get started with Writers' Toolkit
+
+This guide provides includes everything you need to complete your documentation project from start to finish.
+In short, it quickly points you in the right direction.
```

Comment:
Re-reading this, it occurs to me that while we mention it elsewhere in Writer's Toolkit, this would be a good place to add just a sentence or two that mentions markdown and Hugo.  Then line 46 (add `docs/sources`) makes more sense.